src/api/entities/DividendDistribution/index.ts

```diff
@@ -277,27 +285,59 @@ export class DividendDistribution extends CorporateAction {
   }
 
   /**
-   * Retrieve the current amount of withheld tax for a given distribution
+   * Retrieve the amount of taxes that have been withheld up to this point in this Distribution. Optionally, `from` and `to` dates can be passed to retrieve the amount of taxes withheld during a specific period
    *
    * @note uses the middleware
    */
-  public async getWithheldTax(opts: { from?: Date; to?: Date } = {}): Promise<BigNumber | null> {
+  public async getWithheldTax(): Promise<BigNumber> {
     const { id, ticker, context } = this;
 
-    const { from, to } = opts;
-
     const result = await context.queryMiddleware<Ensured<Query, 'getWithholdingTaxesOfCA'>>(
-      getWithholdingTaxesOfCA({
+      getWithholdingTaxesOfCa({
         CAId: { ticker, localId: id.toNumber() },
-        fromDate: from ? from.toISOString().split('T')[0] : null,
-        toDate: to ? to.toISOString().split('T')[0] : null,
+        fromDate: null,
+        toDate: null,
       })
     );
 
-    if (result.data.getWithholdingTaxesOfCA) {
-      return new BigNumber(result.data.getWithholdingTaxesOfCA.taxes);
-    }
+    const withholdingTaxesOfCA = result.data.getWithholdingTaxesOfCA;
```

Comment:
`const { taxes } = result.data.getWithholdingTaxesOfCA!;`

We already confirmed with the harvester team that it can't be null